proto/kvrpcpb.proto

```diff
@@ -386,6 +386,11 @@ message CommitRequest {
     repeated bytes keys = 3;
     // Timestamp for the end of the transaction. Must be greater than `start_version`.
     uint64 commit_version = 4;
+    // primary_key indicates the primary key of the transaction.
+    // If it is empty, it means the primary key is not set and we cannot determine the whether it is the primary commit.
+    // Otherwise if primary_key is present, we can determine the commit is primary or not by checking whether the
+    // primary_key is in the keys.
+    bytes primary_key = 6;
```

Comment:
Another option is to add a field `is_primary_commit`. But I think it may introduce some compatibility issues because `primary_key` is always `false` for clients with old versions.